GUIDELINES.md

```diff
@@ -142,24 +148,27 @@ Add a `make` target
 ### Some things linting will catch
 
 Unused variables, proper casing of function and variable names, undeclared global variables,
-modification of global variables, missing semi-colons. Catching unused variables is great
-because this might alert you to a typo in your code. Modifying global variables could have
-adverse affects on other code within your system, including dependencies, if the other code
-is not aware of the modifications. For these and many other reasons, we think linting is a
+modification of global variables, missing semi-colons, consistent use of const/let/var.
+Catching unused variables is great because this might alert you to a typo in your code.
+Modifying global variables could have adverse affects on other code within your system,
+including dependencies, if the other code is not aware of the modifications. Using `const`
+is preferred to `var`. If a variable must be mutable, use `let` (`var` is also allowed for
+mutable variables for cases where `let` may cause performance issues, but is restricted to
+block scope, so will act like `let`. For these and many other reasons, we think linting is a
```

Comment:
Yeah, docs are hard. I think I would prefer a standard line length (that's not 92 :).